prime-dotnet-webapi/Services/HealthAuthoritySiteService.cs

```diff
@@ -91,6 +91,20 @@ public async Task<IEnumerable<HealthAuthoritySiteAdminListViewModel>> GetSitesAs
         {
             searchOptions ??= new HealthAuthoritySiteSearchOptions();
 
+            int? statusId = null;
+            bool flagged = false;
+            if (searchOptions.StatusId.HasValue)
+            {
+                if (searchOptions.StatusId == 5)
```

Comment:
Instead of these magic numbers, can the C# enum `SiteStatusType` be used instead?

Reply:
Changed accordingly.